entity/entity.h

```diff
@@ -76,11 +76,16 @@ class Entity {
 
   uint32_t GetStencilDepth() const;
 
+  void SetBlendMode(Entity::BlendMode blend_mode);
```

Comment:
Uber minor nit: You don't need `Entity::`

Reply:
Done.